server.py

```
@@ -124,6 +138,83 @@ def _build_knowledge_store(config):
         return None
 
 
+def _build_scheduler(config):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_build_scheduler`

(ANN202)